### PR TITLE
push to pypi

### LINKED ISSUE #362: Publish python library to pypi
This is blocked by #361

### PR DESCRIPTION
**Pull request by pca006132**
closes #362.

(btw I have already pushed as a test)

### CODE REVIEW

**Comment by pca006132** [comment]
PyPI URL: https://pypi.org/project/manifold3d/

@elalish please give me your account handle in PyPI so I can add you as the owner.

**Comment by pca006132** [comment]
tried it just now and it seems to work fine:
```
test@c4239238cc63:/test/manifold$ python3 ./bindings/python/examples/run_all.py 
Traceback (most recent call last):
  File "/test/manifold/./bindings/python/examples/run_all.py", line 33, in <module>
    module = importlib.import_module(f)
  File "/usr/lib/python3.10/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1006, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 688, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/test/manifold/bindings/python/examples/bricks.py", line 17, in <module>
    from manifold3d import Manifold
ModuleNotFoundError: No module named 'manifold3d'
test@c4239238cc63:/test/manifold$ pip3 install manifold3d
Defaulting to user installation because normal site-packages is not writeable
Collecting manifold3d
  Downloading manifold3d-2.2.0-cp310-cp310-manylinux_2_28_x86_64.manylinux_2_27_x86_64.whl (1.1 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 1.1/1.1 MB 14.1 MB/s eta 0:00:00
Requirement already satisfied: numpy in /home/test/.local/lib/python3.10/site-packages (from manifold3d) (1.26.1)
Installing collected packages: manifold3d
Successfully installed manifold3d-2.2.0
test@c4239238cc63:/test/manifold$ python3 ./bindings/python/examples/run_all.py 
Took 14.0ms for bricks
Took 23.0ms for cube_with_dents
Took 0.4ms for extrude
Took 877.4ms for maze
Took 1.8ms for sponge
Took 0.2ms for union_failure
```

**Comment by elalish** [comment]
Okay, I got myself a PyPI account - I'm elalish as usual. I'm also almost to the bottom of #579 - hoping to have a fix today or tomorrow. Then we should do a proper release across Github, npm, and PyPI. Sound good?

**Comment by pca006132** [comment]
Yeah, we can merge this and then manually do a release later. The action will be automagically invoked when we create a release. We have to manually check the version to make sure they are consistent though.

I will add you to the PyPI project tmr.

File: .github/workflows/build_wheels.yml [hunk 1]
```diff
@@ -22,3 +22,35 @@ jobs:
...
+        submodules: true  # Optional, use if you have submodules
+
+    - name: Build SDist
+      run: pipx run build --sdist
+
+    - uses: actions/upload-artifact@v3
+      with:
+        path: dist/*.tar.gz
+
+  pypi-publish-beta:
```
Comment (elalish) [inline .github/workflows/build_wheels.yml:41]: beta until we know it works?

**Review by elalish — APPROVED** [review]
Excellent, thanks!

File: .github/workflows/build_wheels.yml (hunk 1 above)
Comment (pca006132) [reply .github/workflows/build_wheels.yml]: oh I just forgot to change the name

**Pull request merged**